Improvement/arsn 267 support updaterole action

williamlardier opened the pull request:
Support UpdateRole. Needed (maybe) for https://github.com/scality/Vault/pull/2074.

Also fixed a tets failure introduced by a new ubuntu-latest image using the NodeJS version 16.17.0 that introduced a regression, it seems.

Seragonia committed:
ARSN-267: support UpdateRole action

Seragonia committed:
ARSN-267: bump project version

bert-e commented:
# Hello williamlardier,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/arsenal/doc/user).





*Status report is not available.*

bert-e commented:
# Conflict




A conflict has been raised during the creation of
integration branch `w/8.1/improvement/ARSN-267-support-updaterole-action` with contents from `improvement/ARSN-267-support-updaterole-action`
and `development/8.1`.

**I have not created the integration branch.**


Here are the steps to resolve this conflict:

```bash
 $ git fetch
 $ git checkout -B w/8.1/improvement/ARSN-267-support-updaterole-action origin/development/8.1
 $ git merge origin/improvement/ARSN-267-support-updaterole-action
 $ # <intense conflict resolution>
 $ git commit
 $ git push -u origin w/8.1/improvement/ARSN-267-support-updaterole-action
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

Seragonia committed:
ARSN-267: fix failing unit test

NodeJS 16.17.0 introduced a change in the error handling of TLS sockets
in case of error. The connexion is closed before the response is sent,
so handling the ECONNRESET error in the affected test will unblock it,
until this is fixed by NodeJS, if appropriate.

bert-e commented:
# History mismatch



Merge commit #4387c185a7f97ac93d40b229329a32c99c9762a0 on the integration branch
`w/8.1/improvement/ARSN-267-support-updaterole-action` is merging a branch which is neither the current
branch `improvement/ARSN-267-support-updaterole-action` nor the development branch
`development/8.1`.

It is likely due to a rebase of the branch `improvement/ARSN-267-support-updaterole-action` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**

ghost commented:
/reset

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.

bert-e commented:
# Conflict




A conflict has been raised during the creation of
integration branch `w/8.1/improvement/ARSN-267-support-updaterole-action` with contents from `improvement/ARSN-267-support-updaterole-action`
and `development/8.1`.

**I have not created the integration branch.**


Here are the steps to resolve this conflict:

```bash
 $ git fetch
 $ git checkout -B w/8.1/improvement/ARSN-267-support-updaterole-action origin/development/8.1
 $ git merge origin/improvement/ARSN-267-support-updaterole-action
 $ # <intense conflict resolution>
 $ git commit
 $ git push -u origin w/8.1/improvement/ARSN-267-support-updaterole-action
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

ghost commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/7.10`

* :heavy_check_mark: `development/8.1`



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.4`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/7.10`

* :heavy_check_mark: `development/8.1`



The following branches have **NOT** changed:

* `development/6.4`
* `development/7.4`



Please check the status of the associated issue ARSN-267.

Goodbye williamlardier.
